src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -28,11 +29,15 @@ const controlTypeToButtonType = {
 function renderButton(action, display, style, showText, asItem) {
   const {
     icon,
+    ID,
     type,
     text,
     tooltip,
   } = display;
   const isDisabled = display.state === MeetingControlState.DISABLED;
+  const isActive = display.state === MeetingControlState.ACTIVE;
+  const isPressed = (ID === ROSTER_CONTROL || ID === SHARE_CONTROL) && isActive;
```

Comment:
Why are you hard coding specifically for this buttons? how would this work if we add another button that should behave this way? This is not scalable